app/controllers/api/v1/cards_controller.rb

```diff
@@ -42,17 +42,16 @@ def update
   
   # DELETE /api/v1/cards/1
   def destroy
+    card_json = CardSerializer.new(@card).as_json
     @card.destroy
-    render json: { success: true }, status: :ok
+    render json: { card: card_json }, status: :ok
```

Comment:
Now it makes sense to do `if-else` on destroy and send `errors` hash to client when there is an issue with destroying the object.